cli.ts

```diff
@@ -368,7 +368,7 @@ const loadInternalTasks = (cliConfig: CLIConfig, config: LoadedConfig.t, env: En
 		handler: (parsedArgs: SanitizedArgs.t) =>
 			pipe(
 				SanitizedArgs.ofInstallTaskArgs(parsedArgs),
-				chain(args => NPM.installPlugin(config, parsedArgs.projectDir, i18n, args.pluginName)),
+				chain(args => NPM.installPlugin(config, config.projectDir, i18n, args.pluginName)),
```

Comment:
why is `parsedArgs` changed to `config` all of a sudden (here and in other places)?

Reply:
The projectDir from parsedArgs represents the value provided by the user - but the actual project directory might be several traversals up. So the parsedArgs.projectDir is used to represent the user input value, and the config.projectDir is used to represent the actual project root.